fixtures/stairville/mh-x50.json

```diff
@@ -88,31 +88,38 @@
         },
         {
           "type": "Gobo",
-          "name": "Gobo 2"
+          "name": "Gobo 2",
```

Comment:
You can drop the gobo names here, then they will inherit the name from the resource, which is more helpful than just a number.